src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -641,12 +641,13 @@ function translateCellDisplayOutput(output: NotebookCellOutput): JupyterOutput {
         }
         case 'stream': {
             const outputs = output.outputs
-                .filter((opit) => opit.mime === CellOutputMimeTypes.textStream)
+                .filter((opit) => opit.mime === CellOutputMimeTypes.stderr || opit.mime === CellOutputMimeTypes.stdout)
                 .map((opit) => opit.value as string | string[])
                 .reduceRight<string[]>(
                     (prev, curr) => (Array.isArray(curr) ? prev.concat(...curr) : prev.concat(curr)),
                     []
                 );
+
             result = {
```

Comment:
Minor comment here on the result. The OutputItems now have a mimeType that indicates if they are stderr or stdout. Given that mimeType it's not 100% needed that we still have the streamName in the customMetadata (it's what we previously used to determine this). However I have kept it in. Having the streamName on the Output (a level up from the OutputItems[]) is convenient for something like this where we can just look at it to reconstitute the IStream item from the single Output. Jupyter creates different outputs for stderr and stdout, so you don't have to worry about a case where there are mixed stderr and stdout stream mimetypes in the array of OutputItems.

If we don't like this I could remove the streamName property from metadata and just look at the mimeType of the first item in the output array.

Reply:
> If we don't like this I could remove the streamName property from metadata and just look at the mimeType of the first item in the output array.

Yes thats a good idea, I think we should do that as its cleaner, less crud to carry around.

Reply:
I.e. please remove the property `streamName`.

Reply:
Yeah, I'm thinking the same. Will do.